### PR TITLE
pipenv installation instructions

### PR DESCRIPTION
**Pull request by Bost**

### CODE REVIEW

**Comment by gribok** [comment]
Can you please add `pipenv` command to install dev requirements as well? 
See #204

**Comment by Bost** [comment]
Added.

File: README.md [hunk 1]
```diff
@@ -408,11 +415,19 @@ You will need the following things properly installed on your computer.
 
 ### Running Tests
 
-* `make test`
+```bash
+pipenv shell
+pipenv install --dev
```
Comment (Kilo59) [inline README.md]: This really should be `pipenv sync --dev`.
`install` will re-resolve all the dependencies if packages have been updated, potentially leading to a problem. 
`sync` installs whatever is in `Pipfile.lock`.
https://pipenv.pypa.io/en/latest/advanced/#using-pipenv-for-deployments

File: README.md [hunk 2]
```diff
@@ -398,6 +398,13 @@ You will need the following things properly installed on your computer.
 
 * `git clone https://github.com/ExpDev07/coronavirus-tracker-api.git`
 * `cd coronavirus-tracker-api`
+
+Now, make sure you're running python 3.8 and that your `pipenv` actually uses it! It may use packages from `$HOME/.local/lib/python3.7` even when started with `pipenv --python 3.8`. To combat this, (re)install it using:
+```
+/path/to/python3.8 -m pip install pipenv
```
Comment (Kilo59) [inline README.md]: A better practice would be to use `pipx` or `brew` to install `pipenv`
https://pipxproject.github.io/pipx/

File: tests/conftest.py [hunk 3]
```diff
@@ -16,3 +16,4 @@ def api_client():
     The test client uses the requests library for making http requests.
     """
     return TestClient(APP)
+
```
Comment (Kilo59) [inline tests/conftest.py]: We now require that the code be formatted with the `black` code formatter.
That's why the build is failing.
You can do `pipenv run fmt` if your branch is up to date.
or `black . -l 120` directly

File: README.md (hunk 2 above)
Comment (Bost) [reply README.md]: But then you need to install `pipx` in the first place, which is yet another thing which can go wrong. So when having the choice - to get X done:
1. you need to have Y.
2. just use tools you already have.

I prefer the 2., unless given sufficiently strong arguments for 1.

Anyway I'm not much of a python developer, and I don't really want to become one, so it's up to you - change it the way you like. That applies to both or your remarks.

File: tests/conftest.py (hunk 3 above)
Comment (Bost) [reply tests/conftest.py]: Ups, that was a mistake. I didn't want to change the `tests/conftest.py`. Sorry

File: README.md (hunk 2 above)
Comment (Kilo59) [reply README.md]: Arguments about the best way to install `pipenv` aside...
What you've written here is factually incorrect.
You don't need to re-install `pipenv` in order to have it create a `3.8` virtual environment.
`pipenv` installed via `3.6` can still create and use a `3.8` environment as long as `3.8` is installed and on the `PATH`
https://pipenv.readthedocs.io/en/latest/basics/#specifying-versions-of-python

You don't need to include detailed instructions about `brew` or `pipx` installations.
Something simple with links to more detailed instructions should be sufficient.

1. Make sure you have [`python3.8` installed and on your `PATH`](https://docs.python-guide.org/starting/installation/).
2. [Install the `pipenv` dependency manager](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv)
   * with pip/pip3 directly `$ pip install --user pipenv`
   * with Homebrew/Linuxbrew `$ brew install pipenv`
   *  with [pipx](https://pipxproject.github.io/pipx/) `$ pipx install pipenv`

**Comment by Bost** [comment]
> What you've written here is factually incorrect.

Whatever. Correct me as you please. Write there what you want and need.
As I said, I'm really not interested it these things, the same way e.g. I'm not interested in manually changing gears of my car. All I want is to move forward, backward, left or right, slower or faster. I'm not interested in pushing or pulling any levers, strings, buttons pedals whatever. In fact I'm not even interested in cars. All I want to get from A to B. In this case get productive again, get my dev environment running. Thanks.

**Pull request merged**